components/ibm-components/spark/store_spark_model/src/store_spark_model.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+import json
+import os
+import re
+from pyspark.sql import SparkSession
+from pyspark.ml.pipeline import PipelineModel
+from pyspark import SparkConf, SparkContext
+from pyspark.ml import Pipeline, Model
+from watson_machine_learning_client import WatsonMachineLearningAPIClient
+from minio import Minio
+
+
+def get_secret_creds(path):
+    with open(path, 'r') as f:
+        cred = f.readline().strip('\'')
+    f.close()
+    return cred
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--bucket_name', type=str, help='Object storage bucket name', default="dummy-bucket-name")
+    parser.add_argument('--model_filepath', type=str, help='Name of the trained model zip', default="model.zip")
+    parser.add_argument('--pipeline_filepath', type=str, help='Name of the spark pipeline zip', default="pipeline.zip")
```

Comment:
Need to make this more descriptive - potential confusion vis a vis pipeline

Reply:
removed this parameter since we found another way to load the pipeline from the model's metadata.